includes/class-ckwc-order.php

```diff
@@ -890,16 +907,11 @@ private function custom_field_data( $order ) {
 		 *
 		 * @since   1.4.3
 		 *
-		 * @param   mixed                       $fields     Custom Field Key/Value pairs (false | array).
+		 * @param   array                       $fields     Custom Field Key/Value pairs (false | array).
 		 * @param   WC_Order|WC_Order_Refund    $order      WooCommerce Order.
 		 */
 		$fields = apply_filters( 'convertkit_for_woocommerce_custom_field_data', $fields, $order );
 
-		// If the fields array is empty, no Custom Field mappings exist.
```

Comment:
This is deliberate; the WordPress Libraries expects a blank array, not `false`, if there are no custom fields.